File: src/models/player.js
module.exports = function (sequelize, Sequelize) {
    var player = sequelize.define('player', {
        fifaDisplayName: {
            type: Sequelize.STRING,
        },
        firstName: {
            type: Sequelize.STRING,
        },
        lastName: {
            type: Sequelize.STRING,
        },
        shirtName: {
            type: Sequelize.STRING,
        },
        dateOfBirth: {
            type: Sequelize.STRING,
        },
        number: {
            type: Sequelize.INTEGER,
        },
        height: {
            type: Sequelize.INTEGER,
        }
    }, {
        classMethods: {
            associate: function (models) {
                player.belongsTo(models.country, {
                    foreignKey: 'countryId',
                });
                player.belongsTo(models.club, {
                    foreignKey: 'clubId',
                });
                player.belongsTo(models.position, {
                    foreignKey: 'positionId',
                });
            }
        },
        timestamps: false,
    });

    return player;
};

File: models/player.py
from website import db


class Player(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    fifaDisplayName = db.Column(db.Text)
    firstName = db.Column(db.Text)
    lastName = db.Column(db.Text)
    shirtName = db.Column(db.Text)
    dateOfBirth = db.Column(db.Text)
    number = db.Column(db.Integer)
    height = db.Column(db.Integer)

    country_id = db.Column(db.Integer, db.ForeignKey('country.id'))
    position_id = db.Column(db.Integer, db.ForeignKey('position.id'))
    club_id = db.Column(db.Integer, db.ForeignKey('club.id'))
